lib/src/index.json

```diff
@@ -138,16 +138,16 @@
             "to": "./Tabs/index.js"
         },
         {
-            "from": "./src/TextLabel/index.ts",
-            "to": "./TextLabel/index.js"
+            "from": "./src/TextArea/index.ts",
+            "to": "./TextArea/index.js"
         },
         {
-            "from": "./src/Textarea/index.ts",
-            "to": "./Textarea/index.js"
+            "from": "./src/TextBox/index.ts",
+            "to": "./TextBox/index.js"
         },
         {
-            "from": "./src/Textbox/index.ts",
-            "to": "./Textbox/index.js"
+            "from": "./src/TextLabel/index.ts",
+            "to": "./TextLabel/index.js"
```

Comment:
The names have been changed from
- `TextArea` ➡ `Textarea`
- `TextBox` ➡ `Textbox`

Make sure it reflects that in your branch

Reply:
updated